rotkehlchen/types.py

```diff
@@ -319,6 +319,25 @@ class SupportedBlockchain(Enum):
     AVALANCHE = 'AVAX'
     POLKADOT = 'DOT'
 
+    @staticmethod
+    def from_value(value: str) -> 'SupportedBlockchain':
```

Comment:
you don't need this. If you do `SupportedBlockchain('AVAX')` or `SupportedBlockchain('ETH')` etc. it should work. So this function can go away.

Reply:
nice catch :+1: